src/components/CallView/shared/Screen.vue

```diff
@@ -22,6 +22,7 @@
 	<div :id="screenContainerId" class="screenContainer">
 		<video v-show="(localMediaModel && localMediaModel.attributes.localScreen) || (callParticipantModel && callParticipantModel.attributes.screen)"
 			ref="screen"
+			:disablePictureInPicture="!isBig"
```

Comment:
So here you insert a boolean while in `src/components/CallView/shared/LocalVideo.vue` its a string?

Reply:
if I remove the ":" then the rendered result will be `<video ... disablePictureInPicture="!isBig"` so I have to keep it so the variable gets evaluated

in LocalVideo I didn't add the ":" because `<video>` is not a Vue component so "true" ends up rendered as string anyway

Reply:
then
```suggestion
			:disablePictureInPicture="!isBig ? 'true' : 'false'"
```
?

Reply:
your proposal only makes sense if isBig can be null or undefined in which case it would become the string "null" or "undefined".

anyway, better safe than sorry, I take it